tornado/ioloop.py

```diff
@@ -600,7 +601,11 @@ def call_later(
         return self.call_at(self.time() + delay, callback, *args, **kwargs)
 
     def call_at(
-        self, when: float, callback: Callable[..., None], *args: Any, **kwargs: Any
+        self,
+        when: float,
+        callback: Callable[..., Optional[Awaitable]],
```

Comment:
`call_at` and `call_later` should have the same type signature. And I think that since they don't do anything with the return value of their argument, they should actually follow the example of `add_callback` and take an unspecialized `Callable`. (or equivalently, `Callable[..., Any]`, which is what asyncio uses)